docs/src/main/sphinx/connector/hive-alluxio.rst

```diff
@@ -1,91 +1,21 @@
-``hive.config.resources`` connector property.
-
-Deploy Alluxio with Trino
---------------------------
-
-To achieve the best performance running Trino on Alluxio, it is recommended
-to collocate Trino workers with Alluxio workers. This allows reads and writes
-to bypass the network (*short-circuit*). See `Performance Tuning Tips for Presto with Alluxio
-<https://www.alluxio.io/blog/top-5-performance-tuning-tips-for-running-presto-on-alluxio-1/?utm_source=trino&utm_medium=trinodocs>`_
-for more details.
-
-.. _alluxio-catalog-service:
-
-Alluxio catalog service
------------------------
-
-An alternative way for Trino to interact with Alluxio is via the
-`Alluxio catalog service <https://docs.alluxio.io/os/user/stable/en/core-services/Catalog.html?utm_source=trino&utm_medium=trinodocs>`_.
-The primary benefits for using the Alluxio catalog service are simpler
-deployment of Alluxio with Trino, and enabling schema-aware optimizations
-such as transparent caching and transformations. Currently, the catalog service
-supports read-only workloads.
-
-The Alluxio catalog service is a metastore that can cache the information
-from different underlying metastores. It currently supports the Hive metastore
-as an underlying metastore. In order for the Alluxio catalog to manage the metadata
-of other existing metastores, the other metastores must be "attached" to the
-Alluxio catalog. To attach an existing Hive metastore to the Alluxio
-catalog, simply use the
-`Alluxio CLI attachdb command <https://docs.alluxio.io/os/user/stable/en/operation/User-CLI.html?utm_source=trino&utm_medium=trinodocs#attachdb>`_.
-The appropriate Hive metastore location and Hive database name need to be
-provided.
-
-.. code-block:: text
-
-    ./bin/alluxio table attachdb hive thrift://HOSTNAME:9083 hive_db_name
-
-Once a metastore is attached, the Alluxio catalog can manage and serve the
-information to Trino. To configure the Hive connector for Alluxio
-catalog service, simply configure the connector to use the Alluxio
-metastore type, and provide the location to the Alluxio cluster.
-For example, your ``etc/catalog/alluxio.properties`` should include
-the following:
-
-.. code-block:: text
-
-    connector.name=hive
-    hive.metastore=alluxio-deprecated
-    hive.metastore.alluxio.master.address=HOSTNAME:PORT
-
-Replace ``HOSTNAME`` with the Alluxio master hostname, and replace ``PORT``
-with the Alluxio master port.
-An example of an Alluxio master address is ``master-node:19998``.
-Now, Trino queries can take advantage of the Alluxio catalog service, such as
-transparent caching and transparent transformations, without any modifications
-to existing Hive metastore deployments.
+For information on how to setup, configure, and use Alluxio, refer to `Alluxio's
+documentation on using their platform with Trino
+<https://docs.alluxio.io/ee/user/stable/en/compute/Trino.html?utm_source=trino&utm_medium=trinodocs>`_.
```

Comment:
Why does L10 use `https://docs.alluxio.io/os/user` and this line uses `https://docs.alluxio.io/ee/user`? (os vs ee)

Reply:
Good question..

Reply:
Seems like the  pages are the same and it just affects the drop down on the right ..